src/content/docs/en/release/41-upgrades.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: Early Upgrades for Ultramarine 41
+description: "A temporary notice for the 39 to 40 upgrade path."
+---
+
+Like with Ultramarine 40, we're offering early upgrades from Ultramarine 40 to 41 ahead of image release.
+
+Fedora 41 is out, and Ultramarine 41's features are largely focused around the installation experience. Features outside of these are stable enough for most people to upgrade
+
+## Upgrading
+
+Open your Edition's app store, on Flagship, GNOME, and Xfce, this is Software, on Plasma this is Discover. The upgrade should be in the updates section, simply press upgrade, wait for the packages to download, and reboot from the app store.
```

Comment:
```suggestion
Open your Edition's app store ("Software" on Flagship, GNOME, and Xfce; "Discover" on Plasma). The upgrade should be in the updates section, simply press upgrade, wait for the packages to download, and reboot from the app store.
```

Reply:
neither of our options are very clear, i can just try again here